packages/slate-theme/src/sections/header.liquid

```diff
@@ -78,31 +78,41 @@
   </header>
 
   <nav role="navigation">
-    <ul class="site-nav">
+    <ul>
       {% for link in linklists[section.settings.main_linklist].links %}
-        {%- assign child_list_handle = link.title | handleize -%}
-        {% if menus[child_list_handle].links != blank %}
-          <li class="site-nav--has-submenu{% if link.active %} site-nav--active{% endif %}">
-            <a href="{{ link.url }}" class="site-nav__link">
-              {{ link.title }}
-              {% include 'icon-arrow-down' %}
-            </a>
-            <ul class="site-nav__submenu">
-              {% for childlink in menus[child_list_handle].links %}
-                <li {% if childlink.active %}class="site-nav--active"{% endif %}>
-                  <a href="{{ childlink.url }}" class="site-nav__link">{{ childlink.title | escape }}</a>
-                </li>
+        {% if link.links != blank %}
+          <li>
```

Comment:
Yeah good point - I was on the fence about this.

I figured because it's super boilerplate, and we're not leveraging any additional classes it stood out more. We're doing something similar on Narrative:

```liquid
<li class="nav__sublink{% if childlink.links != blank %} nav__has-sublinks{% endif %}">
```

Syntactically it ends up being personal preference once you implement a third level - I'll leave this up to you and @t-kelly.